modin/engines/base/axis_partition.py

```diff
@@ -2,7 +2,7 @@
 from modin.data_management.utils import split_result_of_axis_func_pandas
 
 
-class BaseAxisPartition(object):
+class BaseAxisPartition(object):  # pragma: no cover
```

Comment:
Are we skipping this class due to the abstract functions? If so, can there be a `raise NotImplementedError` to `.convergerc` similar to the example [here](https://coverage.readthedocs.io/en/latest/config.html#syntax)

Reply:
Yes, it is because of the abstract methods. 

Sure adding that makes sense.